lua/entities/gmod_wire_lever.lua

```diff
@@ -58,18 +58,21 @@ if CLIENT then
 
 		local isClicking = LocalPlayer():KeyDown(IN_USE) or LocalPlayer():KeyDown(IN_ATTACK)
 		if isClicking and not self.wasClicking and IsValid(self.csmodel) then
-			local rayPos = util.IntersectRayWithOBB(
-				LocalPlayer():GetShootPos(),
-				LocalPlayer():GetEyeTrace().Normal * 75,
-				self.csmodel:GetPos(),
-				self.csmodel:GetAngles(),
-				self.csmodel:OBBMins(),
-				self.csmodel:OBBMaxs()
-			)
-			if rayPos then
-				net.Start("wire_lever_activate")
-					net.WriteEntity(self)
-				net.SendToServer()
+			local aimPos = LocalPlayer():GetShootPos()
+			if aimPos:DistToSqr(self:GetPos())<160^2 then
+				local rayPos = util.IntersectRayWithOBB(
+					aimPos,
+					LocalPlayer():GetAimVector() * 160,
```

Comment:
I think this might be a bit far - 70-100 felt ok, but I tried 150 and kept mishitting the lever when trying to use nearby buttons/other-levers.